etc/doc/tutorial/01.2-Exploring-the-Interface.md

```diff
@@ -94,7 +94,7 @@ pictures with sound (https://en.wikipedia.org/wiki/Lissajous_curve).
 
 All internal and external events (called cues in Sonic Pi) are
 automatically logged in the Cue Viewer. For example, if you have a MIDI
-controller connected and a press one of its button, youll see a new cue
+controller connected and a press one of its buttons, you'll see a new cue
```

Comment:
s/and a press/and press ?

Reply:
and press one of its buttons, you'll :-)